uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/api/v3/DocumentTest.php

```diff
@@ -170,4 +171,65 @@ function testCreateDocumentClone() {
     // cloned document.
     $this->assertEquals(0, $clonedCount['values']);
   }
+
+  /**
+   * Only documents with "remind me" = true should be cloned
+   */
+  public function testRemindMeDocumentClone() {
+    civicrm_api3('TASettings', 'set', [
+      'fields' => ['days_to_create_a_document_clone' => 5],
+    ]);
+
+    $contact = civicrm_api3('Contact', 'create', [
+      'contact_type' => 'Individual',
+      'display_name' => 'First Contact',
+      'email' => 'test1@email3215155.com',
+    ]);
+    $details1 = 'First sample document details';
+    $details2 = 'Second sample document details';
+    $today = (new DateTime())->format('Y-m-d');
+    $tomorrow = (new DateTime('tomorrow'))->format('Y-m-d');
```

Comment:
if you're not really using the `DateTime` instances, I think this will be simpler with the `date()` function